src/sql/azdata.d.ts

```diff
@@ -3113,7 +3113,7 @@ declare module 'azdata' {
 		removeFormItem(formComponent: FormComponent | FormComponentGroup): boolean;
 	}
 
-	export interface Component extends ComponentProperties {
+	export interface Component extends ComponentProperties, vscode.Disposable {
```

Comment:
Put this in proposed.d.ts (you just create a new interface definition that only extends this. They get combined at compile time)